samples/snippets/subscriber.py

```diff
@@ -524,89 +518,63 @@ def synchronous_pull_with_lease_management(project_id, subscription_id):
     # [START pubsub_subscriber_sync_pull_with_lease]
     import logging
     import multiprocessing
-    import random
+    import sys
     import time
 
     from google.cloud import pubsub_v1
 
+    multiprocessing.log_to_stderr()
```

Comment:
Do you want to show these logging setup in our documentation?

Reply:
Yes. It makes sense for this sample. The idea is to pull messages synchronously, take one and process them in a different process (or thread), extend each message's acknowledgement deadline as it is being processed. Logging here helps user keep track of which process is handling which message.

Reply:
Thanks